src/traces/contour/attributes.js

```diff
@@ -106,7 +107,33 @@ module.exports = extendFlat({}, {
             role: 'style',
             description: [
                 'Determines whether or not the contour lines are drawn.',
-                'Has only an effect if `contours.coloring` is set to *fill*.'
+                'Has an effect only if `contours.coloring` is set to *fill*.'
+            ].join(' ')
+        },
+        showlabels: {
+            valType: 'boolean',
+            dflt: false,
+            role: 'style',
+            description: [
+                'Determines whether to label the contour lines with their values.'
+            ].join(' ')
+        },
+        font: extendFlat({}, fontAttrs, {
```

Comment:
-> `labelFont` in https://github.com/plotly/plotly.js/pull/1815/commits/2116a124eb68aa3f8e7ae051d36bf03862f72afc